test/e2e/pipeline/installer/liqoctl/unpeer.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+for i in $(seq 1 "${CLUSTER_NUMBER}");
+do
+  export KUBECONFIG="${TMPDIR}/kubeconfigs/liqo_kubeconf_${i}"
+
+  for foreignCluster in $(kubectl get foreignclusters.discovery.liqo.io --no-headers -o custom-columns=":metadata.name");
+  do
+    "${LIQOCTL}" remove cluster "${foreignCluster}"
```

Comment:
It is probably necessary to disable also the discovery, to avoid rediscovering the clusters after some time.